x/simulation/rand_util.go

```diff
@@ -88,6 +88,43 @@ func RandIntBetween(r *rand.Rand, min, max int) int {
 	return r.Intn(max-min) + min
 }
 
+// returns random subset of the provided coins
+// will return at least one coin unless coins argument is empty or malformed
+// i.e. 0 amt in coins
+func RandSubsetCoins(r *rand.Rand, coins sdk.Coins) sdk.Coins {
```

Comment:
I want to use this for `RandomFees` as well so fees can be paid in multiple coins rather than just one denom. Thoughts?

Reply:
Yeah I think that's fine.